### PR TITLE
add Gradle to getting-started

### PR DESCRIPTION
**Pull request by vorburger**
#75

### CODE REVIEW

**Comment by vorburger** [comment]
@jnizet

**Comment by vorburger** [comment]
This is fully ready for review and merge from my side now.

@stalep @geoand @jnizet would you like to review?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,6 +54,16 @@ mvn clean verify -Pnative
...
 
+## Build with Gradle
+
+Clone like above, and build e.g. `getting-started` using:
+
+    ../gradlew build
+
+You'll find the `*-runner.jar` in `build` (not `build/libs`).
+
+We may change it to run on the regular "build" task instead of "build-shamrock" later.
```
Comment (stalep) [inline CONTRIBUTING.md]: The `build` task automatically runs `quarkusBuild` so this can be removed I think.

File: getting-started/build.gradle [hunk 2]
```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'java'
+    id 'io.quarkus.gradle.plugin' version '0.12.0'
```
Comment (stalep) [inline getting-started/build.gradle]: The Maven quickstarts use `999-SNAPSHOT` as the version for it's quickstarts. I think we should do similar? @cescoffier  ?

File: getting-started/build.gradle [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    }
+}
+*/
+repositories {
+     mavenCentral()
+     maven { url 'http://repository.jboss.org/nexus/content/groups/public' }
+}
+
+dependencies {
+    compileOnly group: 'io.quarkus', name: 'quarkus-resteasy', version:'0.12.0'
```
Comment (stalep) [inline getting-started/build.gradle]: We've changed this to from `compileOnly`to `implementation` in the docs. Also the version.

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-bin.zip
```
Comment (stalep) [inline gradle/wrapper/gradle-wrapper.properties]: In master we use 5.3 now.

File: CONTRIBUTING.md (hunk 1 above)
Comment (vorburger) [reply CONTRIBUTING.md]: Oups, yeah, of course, well spotted.. line 61 used to be `gradlew quarkusBuild` and I changed but didn't remove this.

File: getting-started/build.gradle (hunk 2 above)
Comment (vorburger) [reply getting-started/build.gradle]: @stalep erm... sorry, but I don't think that's true? There is no `999-SNAPSHOT` and [the Maven quickstarts uses fixed version](https://github.com/quarkusio/quarkus-quickstarts/blob/master/getting-started/pom.xml#L10) (click link). If you agree, will you Resolve Conversation on this point?

File: gradle/wrapper/gradle-wrapper.properties (hunk 4 above)
Comment (vorburger) [reply gradle/wrapper/gradle-wrapper.properties]: done!

File: getting-started/build.gradle (hunk 3 above)
Comment (vorburger) [reply getting-started/build.gradle]: Sure, done.

File: getting-started/build.gradle (hunk 2 above)
Comment (stalep) [reply getting-started/build.gradle]: ah... my fault! I have a local commit where i have changed it to 999-SNAPSHOT, sorry ignore this

**Comment by vorburger** [comment]
> looks fine, please just squash the commits before we merge :)

done

**Pull request merged**